### ISSUE TITLE
Is there a way to get the Window Size/Position created by InspectorPlugin?

### ISSUE THREAD

**Issue by cole21771** [jakobhellermann/bevy-inspector-egui#108]
A part of my game involves doing something on clicking anywhere in the game.

However, clicks on the egui inspector window count as clicks in the game and are caught by my clicking system. I'd love to find a way to exclude clicks within the window from getting picked up by that system. 

I thought maybe I could get the coordinates of the window and it's size and do some filtering myself with basic math, but I haven't been able to find a way to get the egui window coords or window size created by the InspectorPlugin. Am I missing something or is there no way to get that data?

**Comment by cole21771** [comment]
Thank you! That worked exactly as I had hoped with this simple code
```rust
let hovering_over_egui = egui_context.ctx_for_window_mut(WindowId::default()).is_pointer_over_area();
```
